### ISSUE TITLE
Would be nice to see current directory in the commit dialog

### ISSUE THREAD

**Issue by GoogleCodeExporter** [guberm/tortoisegit#325]
```
Sometimes after a day of work i do commit several repositories.
Because overlay code right now does not work properly, some of them have no
changes, but you only see that once you are in the commit dialog.
Even though they have no local changes, i still have to push them upstream,
because there may be some unsynced changes from before. To do that i have
to go back to the folder and use push from tortoise git menu, because there
is no push button in the commit dialog itself.
At that point usually for me some of the repositories have been already
pushed (using the button that comes up after commit), but i have no idea
which repositories those are.
So, after i close the commit dialog of the unchanged repository i want to
push, i have no idea which repository this is because there is no
information about that in  the commit dialog.

In short: please add the full path to the repository in question somewhere
(preferably at the very top, above "Commit to: master") of the commit dialog :)



```

Original issue reported on code.google.com by `kipr...@gmail.com` on 24 Feb 2010 at 4:28
